spec/controllers/evidence_controller_spec.rb

```diff
@@ -103,4 +103,19 @@
       end
     end
   end
+
+  describe 'GET #result' do
```

Comment:
Do we need this, if we have feature tests covering the same thing? If we were testing some assignments here, maybe, but we only test response and template, whereas in feature and view model specs we test all aspects of this page.